src/shared/hooks/useToast.ts

```diff
@@ -0,0 +1,30 @@
+import { useEffect } from 'react';
+import { ToastSize, useToastStore } from '@/shared/stores/toastStore';
+import { ToastProps } from '@/shared/stores/toastStore';
+
+const DURATION = 3000;
+
+export const useToast = () => {
+  const { toast, setToast, clearToast } = useToastStore();
+
+  const showToast = (content: string, size: ToastSize) => {
+    const toast: ToastProps = {
+      content,
+      size,
+    };
+
+    setToast(toast);
```

Comment:
이렇게 표현해줘도 괜찮을 것 같습니다~
```suggestion
    setToast({
      content,
      size
    });
```

Reply:
불필요한 과정이었네요 감사합니다!!